src/main/java/com/epam/aidial/core/controller/ControllerSelector.java

```diff
@@ -16,7 +16,7 @@
 @UtilityClass
 public class ControllerSelector {
 
-    private static final Pattern PATTERN_POST_DEPLOYMENT = Pattern.compile("^/+openai/deployments/([^/]+)/(completions|chat/completions|embeddings)$");
+    private static final Pattern PATTERN_POST_DEPLOYMENT = Pattern.compile("^/+openai/deployments/(.+?)/(completions|chat/completions|embeddings)$");
```

Comment:
Reason behind this change? We match raw url. If deployments contains '/' it should be percent encoded.

Reply:
Do lang chain and open ai python libs work with this change?

Reply:
1. we use same technique for resource url
2. we did a brief investigation, and it should work